runtime/sema/check_import_declaration.go

```diff
@@ -149,7 +149,6 @@ func (checker *Checker) importResolvedLocation(resolvedLocation ResolvedLocation
 		checker.valueActivations,
 		resolvedLocation.Identifiers,
 		allValueElements,
-		imp.IsImportableValue,
```

Comment:
As discussed in the meeting; would be good to double check with a test that it is indeed safe to remove the check that these values are importable.